### PR TITLE
parse multiple alleles into distinct Variant records

### PR DESCRIPTION
**Pull request by iskandr**
Fixes https://github.com/hammerlab/varcode/issues/22.

Also fixes bug from the last PEP8 PR which removed `reverse_complement` from `string_helpers`. 

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/varcode/29)

### CODE REVIEW

File: varcode/vcf.py [hunk 1]
```diff
@@ -65,19 +65,21 @@ def load_vcf(
...
-            ref=x.REF,
-            alt=x.ALT[0].sequence,
-            info=x.INFO,
-            ensembl=ensembl)
-        for x in vcf_reader
-        if x.ALT[0] and (not only_passing or not x.FILTER or x.FILTER == "PASS")
-    ]
-
+    variants = []
+    for record in vcf_reader:
```
Comment (timodonnell) [inline varcode/vcf.py]: I suppose this has always been the case for varcode, but this just made me think about it: we're throwing away the genotype information. Even for a regular non-multiallelic variant, I don't think I have a way to see if it was called a het or hom. Would it make sense to retain that in the variant? Or we could potentially address that in a later PR.

**Comment by timodonnell** [comment]
LGTM

File: varcode/vcf.py (hunk 1 above)
Comment (iskandr) [inline varcode/vcf.py]: Yep, currently we're only attaching the INFO dict to variants and not the sample info. What do you think is the right way to handle e.g. genotypes? Should we make a nested sample_name -> key -> value dictionary of dictionaries? That seems messy but I'm not sure what else to do. In any case, that should definitely be a separate PR.
